linux/tools.Dockerfile

```diff
@@ -16,9 +16,14 @@ RUN wget -nv https://azurecliprod.blob.core.windows.net/cloudshell-release/azure
     && dpkg -i azure-cli-latest-buster.deb \
     && rm -f azure-cli-latest-buster.deb
 
+# latest SSHArc 0.2.0 from ClI package.
+RUN wget -nv -q http://aka.ms/SSHArc/CLI/Download \
+    && /opt/ansible/bin/python -m pip install -r ssh-0.2.0-py3-none-any.whl \
+    && rm ssh-0.2.0-py3-none-any.whl
+
 # Install any Azure CLI extensions that should be included by default.
 RUN az extension add --system --name ai-examples -y
-RUN az extension add --system --name ssh -y --version 0.2.0
+RUN az extension add --system --name ssh -y
```

Comment:
By run `docker run -it tools_cloudshell /bin/bash` in the compute-CloudShell, it cannot find the tools-cloudshell.

**Error message:**
`C:\work\compute-CloudShell>docker run -it tools_cloudshell /bin/bash
Unable to find image 'tools_cloudshell:latest' locally
docker: Error response from daemon: pull access denied for tools_cloudshell, repository does not exist or may require 'docker login': denied: requested access to the resource is denied.
See 'docker run --help'.`

I believe this is wrong path in my end, where should I run this command to trigger the docker image of cloud shell tools? Thanks

Reply:
In the local container after build, we have following in the az extension list:
```
[
  {
    "experimental": false,
    "extensionType": "whl",
    "name": "ai-examples",
    "path": "/opt/az/lib/python3.6/site-packages/azure-cli-extensions/ai-examples",
    "preview": true,
    "version": "0.2.5"
  },
  {
    "experimental": false,
    "extensionType": "whl",
    "name": "ssh",
    "path": "/opt/az/lib/python3.6/site-packages/azure-cli-extensions/ssh",
    "preview": true,
    "version": "0.1.8"
  }
]
```

Reply:
I think we want the ssh version to be 0.2.0, not 0.1.8. if you remove the call to 'az extension add --name ssh' hopefully it will have the desired effect?

Reply:
The version 0.2.0 is not published to az cli yet. To install it you need to download the whl file from https://aka.ms/SSHArc/CLI/Download, and then run `az extension add --system --source <path to ssh-0.2.0-py3-none-any.whl>` .